masterlist.yaml

```
@@ -1876,6 +1876,15 @@ plugins:
         subs: [ 'Better Dynamic Snow ESP' ]
         condition: 'many("Better Dynamic Snow( SE)?\.esp")'
   - name: 'Better Dynamic Snow SE.esp'
+    inc:
+      - name: 'SKSE/Plugins/ShaderTools.dll'
+        display: 'SSE Parallax Shader Fix'
+      - name: 'SKSE/Plugins/SSEShaderTools.dll'
+        display: 'SSE Parallax Shader Fix AE'
+    msg:
+      - <<: *useVersion
+        subs: [ '2.11' ]
+        condition: 'file("skse/plugins/(SSE)?\ShaderTools.dll")'
```

Escaped the wrong character 😛 

```suggestion
        condition: 'file("skse/plugins/(SSE)?ShaderTools\.dll")'
```

---

Good catch, thanks!